File: src/domain/models/Customer.ts

export interface Customer {
  id: string;
  brand_name: string;
  contact_name: string;
  contact_email: string;
  contact_phone?: string;
  acquired_services?: string[];
  created_at: string;
  updated_at: string;
}

export interface ServiceUsageData {
  name: string;
  count: number;
}

File: ASP.NET/SurveyApp.Infrastructure/Data/Entities/Customer.cs

using System;
using System.Collections.Generic;

namespace SurveyApp.Infrastructure.Data.Entities
{
    public class Customer
    {
        public int Id { get; set; }
        public string BrandName { get; set; } = string.Empty;
        public string ContactName { get; set; } = string.Empty;
        public string ContactEmail { get; set; } = string.Empty;
        public string? ContactPhone { get; set; }
        public string CustomerType { get; set; } = "client"; // Valor predeterminado
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }

        // Relación con CustomerService
        public ICollection<CustomerService> CustomerServices { get; set; } = new List<CustomerService>();
    }
}
